common/src/IO/IOUtils.cpp

```diff
@@ -97,9 +97,9 @@ namespace TrenchBroom {
             return line.substr(expectedHeader.size());
         }
 
-        void writeGameComment(FILE* stream, const std::string& gameName, const std::string& mapFormat) {
-            std::fprintf(stream, "// Game: %s\n", gameName.c_str());
```

Comment:
I think you can now remove the `cstring` include.

Reply:
Oh, there's still `size_t fileSize(std::FILE* file);` in this header.

Reply:
Ah sorry I misread your comment. Yes, the `cstring` include appears to be unused in this file